wasm/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[workspace]
+resolver = "2"
+members = [
+    "src/sql-lexer-wasm",
+]
+
+[workspace.package]
+edition = "2021"
+rust-version = "1.74.0"
+
+[workspace.metadata.vet]
+store = { path = "../misc/cargo-vet" }
```

Comment:
Mind moving this to `misc/wasm` to avoid cluttering the top level?

Reply:
Sure thing. Moved :heavy_check_mark: